medusa/providers/torrent/html/gimmepeers.py

```diff
@@ -0,0 +1,164 @@
+# coding=utf-8
+
+"""Provider code for GimmePeers."""
+
+from __future__ import unicode_literals
+
+import re
+
+from medusa import logger, tv
+from medusa.bs4_parser import BS4Parser
+from medusa.helper.common import convert_size
+from medusa.providers.torrent.torrent_provider import TorrentProvider
+
+from requests.utils import dict_from_cookiejar
+
+
+class GimmePeersProvider(TorrentProvider):
+    """GimmePeers Torrent provider."""
+
+    def __init__(self):
+
+        TorrentProvider.__init__(self, 'GimmePeers')
+
+        self.urls = {
+            'base_url': 'https://www.gimmepeers.com',
+            'login': 'https://www.gimmepeers.com/takelogin.php',
+            'detail': 'https://www.gimmepeers.com/details.php?id=%s',
+            'search': 'https://www.gimmepeers.com/browse.php',
+            'download': 'https://gimmepeers.com/%s',
+        }
+
+        self.url = self.urls['base_url']
```

Comment:
Get rid of base_url. See how we do this: https://github.com/pymedusa/Medusa/blob/master/medusa/providers/torrent/html/alpharatio.py#L38-L42